src/routes/strings.ts

```diff
@@ -0,0 +1,10 @@
+export const ERROR_ROOM_GET_BY_ID = 'ROOM_GET_BY_ID';
+export const ERROR_ROOM_CREATE_EMPTY_DATA = 'ROOM_CREATE_EMPTY_DATA';
+export const ERROR_ROOM_CREATE_EMPTY_OPTIONS = 'ROOM_CREATE_EMPTY_OPTIONS';
+export const ERROR_ROOM_CREATE_VALIDATION = 'ROOM_CREATE_VALIDATION';
+
+export const ERROR_ROOM_GET_BY_ID_MESSAGE = 'Room does not exists';
```

Comment:
is it better if we store an object instead?

```js
export const ERROR_ROOM_GET_BY_ID = {
  code: 'ROOM_GET_BY_ID',
  description: 'Room does not exists',
}
```

Reply:
Do not like it.  
That approach limits other string to only have code and message properties, but if you include different shapes then the file loses it purpose.

Reply:
but all error messages must be next to an error code, so it makes sense to put them together,  I don't see the case where we want to have different shapes or only strings (on errors context)